### PR TITLE
:art: most advanced code

### PR DESCRIPTION
**Pull request by vikiival**

### CODE REVIEW

File: components/drops/DropCard.vue [hunk 1]
```diff
@@ -102,6 +105,7 @@ const price = computed(() => {
 })
 
 onMounted(async () => {
+  console.log('props.drop.collection', props.drop.collection)
```
Comment (Jarsen136) [inline components/drops/DropCard.vue:108]:
```suggestion
```

**Review by Jarsen136 — COMMENTED** [review]
image

Image dose not load. Otherwise looks nice.

**Comment by vikiival** [comment]
> Image dose not load. Otherwise looks nice.

That is the image ;D

nftstorage.link/ipfs/bafkreidvb2my5vj6jgpbrwa7dfw55tryhfk7um5inuxnz7y7khqse2thjq

**Comment by yangwao** [comment]
uh, okay let's goo we will make design for that 
cc @exezbcz

**Comment by exezbcz** [comment]
@yangwao what is needed 👀

**Comment by vikiival** [comment]
> what is needed 👀



> image
> Image dose not load. Otherwise looks nice.

This is the current link for the form. Wao wants to make it nicer

**Pull request merged**